analytics/models.py

```diff
@@ -49,7 +49,7 @@ class BaseCount(models.Model):
     subgroup = models.CharField(max_length=16, null=True)  # type: Optional[Text]
     end_time = models.DateTimeField()  # type: datetime.datetime
     value = models.BigIntegerField()  # type: int
-    anomaly = models.ForeignKey(Anomaly, null=True)  # type: Optional[Anomaly]
+    anomaly = models.ForeignKey(Anomaly, on_delete=models.CASCADE, null=True)  # type: Optional[Anomaly]
```

Comment:
This should be `on_delete=models.SET_NULL` because anomaly can be `NULL` as well.

Reply:
I see, fixed, thanks!